File: src/server/AnkaCMS.Service/Models/ParameterGroupModel.cs
using AnkaCMS.Core;
using AnkaCMS.Core.ValueObjects;
using System;

namespace AnkaCMS.Service.Models
{
    public class ParameterGroupModel : IServiceModel
    {
        public Guid Id { get; set; }
        public int DisplayOrder { get; set; }
        public bool IsApproved { get; set; }
        public int Version { get; set; }
        public DateTime CreationTime { get; set; }
        public IdCodeName Creator { get; set; }
        public DateTime LastModificationTime { get; set; }
        public IdCodeName LastModifier { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }

    }
}

File: src/client/angular-web-admin/src/app/models/parameter-group-model.ts
import { IdCodeName } from '../value-objects/id-code-name';

export class ParameterGroupModel {
    id: string;
    displayOrder: number;
    isApproved: boolean;
    version: number;
    creationTime: Date;
    creator: IdCodeName;
    lastModificationTime: Date;
    lastModifier: IdCodeName;
    code: string;
    name: string;
    description: string;

    constructor() {
        this.creator = new IdCodeName();
        this.lastModifier = new IdCodeName();
    }
}
